lambdas/db-migration/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import Knex from 'knex';
+import * as path from 'path';
+
+export type Command = 'latest';
+
+export interface HandlerEvent {
+  command?: Command,
+  env?: NodeJS.ProcessEnv
+}
+
+const getRequiredEnvVar = (name: string, env: NodeJS.ProcessEnv): string => {
+  const value = env?.[name];
+
+  if (value) return value;
+
+  throw new Error(`The ${name} environment variable must be set`);
+};
+
+const getConnectionConfig = (env: NodeJS.ProcessEnv): Knex.PgConnectionConfig => ({
+  host: getRequiredEnvVar('PG_HOST', env),
+  user: getRequiredEnvVar('PG_USER', env),
+  // TODO Get this value from secrets manager
+  password: getRequiredEnvVar('PG_PASSWORD', env)
```

Comment:
This should probably require a database. Relying on the default postgres (or other engine default) database feels too inexplicit, particularly for a production environment.

Reply:
c5489c145